### PR TITLE
1 Home Page Hordynskyi

### PR DESCRIPTION
**Pull request by Viktor-Hordynskyi**

### CODE REVIEW

**Review by innomaxx — APPROVED** [review]

**Review by andrii-kondratiuk — APPROVED** [review]

**Review by Dmitry528 — COMMENTED** [review]
Rename all components, first character must be like "MyComponent", not like a "myComponent"

File: Readme.md [hunk 1]
```diff
@@ -1,68 +1,25 @@
...
-### Deployment
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
-
-### `npm run build` fails to minify
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
+<h1>Ukraine 4 You</h1>
+
+**Міні гайд, що потрібно взяти собі, для таких сторінок як:**
```
Comment (katemalash) [inline Readme.md]: Could you, please, explain, what does it mean?

File: Readme.md [hunk 2]
```diff
@@ -1,68 +1,25 @@
...
+* places list page
+* place page
+* my plans page
+* sign in/sign up pages
+* profile page
+
+**шлях до папок:**
```
Comment (katemalash) [inline Readme.md]: The same

File: Readme.md [hunk 3]
```diff
@@ -1,68 +1,25 @@
...
+`U4U_Client/src/components`
+
+**що саме вам потрібно:**
```
Comment (katemalash) [inline Readme.md]: And here

**Review by Dmitry528 — APPROVED** [review]

File: Readme.md (hunk 1 above)
Comment (Viktor-Hordynskyi) [reply Readme.md]: You can review one more time this README.md, it in English now.

**Review by DMTRIP — APPROVED** [review]

**Pull request merged**